test/http-response-message.spec.js

```diff
@@ -1,155 +1,203 @@
 import {HttpResponseMessage, mimeTypes} from '../src/http-response-message';
 import {Headers} from '../src/headers';
 
-describe("HttpResponseMessage", () => {
+function mockXhr(options) {
+  var xhr = {
+    getAllResponseHeaders() { },
```

Comment:
Do we always provide this, and does it make the hack noted in `HttpResponseMessage` unnecessary?